WordPress/twentytwentyfour
# Enable lightbox by default

**Description**

Closes #529 by enabling lightbox on by default for images.

**Testing Instructions**


1. Activate the theme.
2. Add a page.
3. Add an image. 
4. See the "Expand on click" control active by default.
5. Add the "Clients" pattern.
6. Add an image to one of the placeholders. 
7. See "Expand on click" turned off.
